cves/CVE-2012-4387.yml

```diff
@@ -205,4 +231,10 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This was caused by having user inputs be translated to complex expressions with
```

Comment:
Does this mean this was a flaw in the design?

Reply:
I would classify this more as a coding mistake than a design flaw. The approach they opted to take was fine, but they should have been more careful with validation the input they were receiving. Updating my answer to better convey this information.